### PR TITLE
[Documentation][Book][Installation] Upgrading Sylius

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets | |
| License         | MIT |

### CODE REVIEW

File: docs/book/installation/upgrading.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+            "sylius/sylius": "^1.0@beta",
+
+            "...": "...",
+        },
+
+        ...
+    }
+
+**Then run ``composer update`:**
```
Comment (tuka217) [inline docs/book/installation/upgrading.rst]: Should there be two apostrophes?

Comment (TheMadeleine) [inline docs/book/installation/upgrading.rst]: Yes. :D But at the end should be 2 instead of 1. :D

**Comment by pjedrzejewski** [comment]
We are missing migrations command.

File: docs/book/installation/upgrading.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+.. code-block:: bash
+
+    $ composer update sylius/sylius
+
+If this results in a dependency error, it may mean that the also other Sylius dependencies have to be upgraded.
```
Comment (NeverResponse) [inline docs/book/installation/upgrading.rst]: `that the also other` yeah nah
`that other Sylius dependencies also have to be upgraded.` perhaps?

File: docs/book/installation/upgrading.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.. index::
+   single: Upgrading
+
+Upgrading
+=========
+
+Sylius is releasing new versions from time to time. Each release is supported with an UPGRADE file, which is meant to help in the upgrading process,
+especially when the version(major) breaks backwards compatibility.
```
Comment (pamil) [inline docs/book/installation/upgrading.rst]: especially for major versions, which break backwards compatibility. wdyt?

File: docs/book/installation/upgrading.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+**Update the Sylius library via Composer by modifying the composer.json file:**
```
Comment (pamil) [inline docs/book/installation/upgrading.rst]: Sylius library -> Sylius library version constraint?

Comment (pamil) [inline docs/book/installation/upgrading.rst]: Anyway, we can get rid of `via Composer` :)

File: docs/book/installation/upgrading.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    $ composer update sylius/sylius
+
+If this results in a dependency error, it may mean that the also other Sylius dependencies have to be upgraded.
+Using this command may help you upgrade Sylius dependencies.
+
+.. code-block:: bash
+
+    $ composer update sylius/sylius --with-dependencies
+
+If this does not help, it is a matter of debugging the conflicting versions and working out how your composer.json should look after the upgrade.
```
Comment (pamil) [inline docs/book/installation/upgrading.rst]: `composer.json` -> ` ``composer.json`` `?

File: docs/book/installation/upgrading.rst (hunk 3 above)
Comment (TheMadeleine) [inline docs/book/installation/upgrading.rst]: :+1:

**Comment by pjedrzejewski** [comment]
@TheMadeleine Could you rebase, please? Thanks!

**Comment by TheMadeleine** [comment]
@pjedrzejewski Done!

**Comment by pjedrzejewski** [comment]
Thanks Magda! :)

**Pull request merged**